tests/frame_codec_ut/frame_codec_ut.c

```diff
@@ -794,25 +809,23 @@ TEST_FUNCTION(frame_codec_receive_bytes_decodes_one_empty_frame)
 TEST_FUNCTION(frame_codec_receive_bytes_with_not_enough_bytes_for_a_frame_does_not_trigger_callback)
 {
 	// arrange
-	frame_codec_mocks mocks;
 	FRAME_CODEC_HANDLE frame_codec = frame_codec_create(test_frame_codec_decode_error, TEST_ERROR_CONTEXT);
 	(void)frame_codec_subscribe(frame_codec, 0, on_frame_received_1, frame_codec);
-	mocks.ResetAllCalls();
+	umock_c_reset_all_calls();
 	unsigned char frame[] = { 0x00, 0x00, 0x00, 0x08, 0x02, 0x00, 0x00 };
 
-	STRICT_EXPECTED_CALL(mocks, singlylinkedlist_find(TEST_LIST_HANDLE, IGNORED_PTR_ARG, IGNORED_PTR_ARG))
-		.IgnoreArgument(2)
+	STRICT_EXPECTED_CALL(singlylinkedlist_find(TEST_LIST_HANDLE, IGNORED_PTR_ARG, IGNORED_PTR_ARG))
 		.ValidateArgumentBuffer(3, &frame[5], 1);
-	EXPECTED_CALL(mocks, singlylinkedlist_item_get_value(IGNORED_PTR_ARG));
-	EXPECTED_CALL(mocks, singlylinkedlist_item_get_value(IGNORED_PTR_ARG));
-	EXPECTED_CALL(mocks, amqpalloc_malloc(IGNORED_NUM_ARG));
+	EXPECTED_CALL(singlylinkedlist_item_get_value(IGNORED_PTR_ARG));
```

Comment:
Having a framework that also tests the order of the calls is really nice. Thanks for that feature.